### ISSUE TITLE
[DCS] Impossible to create single node instance

### ISSUE THREAD

**Issue by lego963** [opentelekomcloud/terraform-provider-opentelekomcloud#1371]
# Affected Resource(s)
`resource/opentelekomcloud_dcs_instance_v1`

# Expected Behavior
```
    resource_opentelekomcloud_dcs_instance_v1_test.go:24: Step 3/3 error: Error running apply: exit status 1
        
        Error: error creating DCS instance: Missing input for argument [SaveDays]
        
          with opentelekomcloud_dcs_instance_v1.instance_1,
          on terraform_plugin_test.tf line 23, in resource "opentelekomcloud_dcs_instance_v1" "instance_1":
          23: resource "opentelekomcloud_dcs_instance_v1" "instance_1" {
```

**Comment by outcatcher** [comment]
This is actual issue, not acceptance test mistake

**Comment by outcatcher** [comment]
Fixed in #1415
